autode/calculation.py

```diff
@@ -423,16 +424,17 @@ def clean_up(self,
 
         filenames = self.input.filenames
         if everything:
-            filenames.append(self.output.filename)
+            filenames += [fn for fn in os.listdir() if fn.startswith(self.name)]
 
-        logger.info(f'Deleting {filenames}')
+        logger.info(f'Deleting: {set(filenames)}')
 
         # Delete the files that exist
-        for filename in filenames:
-            if not os.path.exists(filename):
+        for filename in set(filenames):
+
+            if os.path.exists(filename):
```

Comment:
better as a try except? (assuming this throws an exception if the file doesn't exist)

Reply:
Yep, probably 👍